framework/src/problems/FEProblemBase.C

```diff
@@ -3787,8 +3820,6 @@ FEProblemBase::solve()
   // sync solutions in displaced problem
   if (_displaced_problem)
     _displaced_problem->syncSolutions();
-
-  Moose::perf_log.pop("solve()", "Execution");
```

Comment:
I thought we were keeping the original perf_log around for awhile.

Reply:
As discussed - I just don't think it's necessary.  We can make this one really good immediately and just move over to it.  It already provides much better info.